packages/devextreme/testing/tests/DevExpress.ui.widgets/progressBar.markup.tests.js

```diff
@@ -136,22 +136,41 @@ QUnit.module('ProgressBar markup', {
 });
 
 QUnit.module('aria accessibility', () => {
-    QUnit.test('aria role', function(assert) {
-        const $element = $('#progressbar').dxProgressBar({});
-        assert.equal($element.attr('role'), 'progressbar', 'aria role is correct');
+    QUnit.test('aria attributes should be correct on init', function(assert) {
+        const $element = $('#progressbar').dxProgressBar({
+            min: 0,
+            max: 50,
+            value: 0
+        });
+
+        assert.equal($element.attr('aria-valuemin'), 0, 'min value');
```

Comment:
It's better always use `strictEqual`
```suggestion
        assert.equal($element.attr('aria-valuemin'), 0, 'min value');
```